pkg/metrics/metrics.go

```diff
@@ -40,14 +37,32 @@ func GetCLIVersion() string {
 	return string(content)
 }
 
-func userIsOptedIn(app *application.Avalanche) bool {
-	if app.Conf.ConfigFileExists() && app.Conf.GetConfigBoolValue(constants.ConfigMetricsEnabledKey) {
-		return true
+func getMetricsUserID(app *application.Avalanche) string {
+	if !app.Conf.ConfigFileExists() || !app.Conf.ConfigValueIsSet(constants.ConfigMetricsUserIDKey) {
+		userID := utils.RandomString(20)
```

Comment:
why are we replacing user id?

Current implementation
`hash := sha256.Sum256([]byte(codespaceName))
userID = base64.StdEncoding.EncodeToString(hash[:])`
		
SHA 256 is irreversible

Reply:
and it eliminates the need to check for app config files

Reply:
legal considers this approach, totally independant of user name for id generation, to be the most
beneficial for the purpose cc @javiertc @learyce

Reply:
checking app config file is not too much of an problem IMO